etl/steps/data/garden/who/2023-10-13/medical_devices_atlas.meta.yml

```diff
@@ -0,0 +1,50 @@
+# NOTE: To learn more about the fields, hover over their names.
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/dataset/
+dataset:
+  update_period_days: 365
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+tables:
+  medical_devices_atlas:
+    variables:
+
+      total_density_per_million_population__computed_tomography_units:
+        title: Total density per million population, computed tomography units
+        unit: units per million population
+        short_unit: ""
+        description_short: Total number of computed tomography units per million population.
+        processing_level: minor
```

Comment:
Nit - If you define this at the YAML file (at the top preferably), it'll be applied to all variables
```
definitions:
  common:
    processing_level: minor
```

Reply:
niceee! :)